src/StatusQ/Components/StatusToastMessage.qml

```diff
@@ -0,0 +1,110 @@
+import QtQuick 2.14
+import QtQuick.Controls 2.12
+import QtQuick.Layouts 1.12
+
+import QtGraphicalEffects 1.13
+import StatusQ.Core 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Core.Theme 0.1
+
+//TODO finish UI
+Control {
+    id: root
+    width: 343
+    height: !!secondaryText ? 68 : 48
+    anchors.right: parent.right
+    anchors.rightMargin: open ? 0 : -width
+    Behavior on anchors.rightMargin { NumberAnimation { duration: 400 }}
+    opacity: open ? 1.0 : 0.0
+    Behavior on opacity { NumberAnimation { duration: 400 }}
+
+    //TODO check on whether to use states
+    //amd remove if not needed
+    property bool open: false
+    property string primaryText: ""
+    property string secondaryText: ""
+
+    function open(title, subTitle) {
+        root.primaryText = title;
+        root.secondaryText = subTitle;
+        root.open = true;
+    }
+
+    signal close()
+
+    background: Rectangle {
+        id: background
+        anchors.fill: parent
+        color: Theme.palette.statusSelect.menuItemBackgroundColor
```

Comment:
Since this is not a select component, we shouldn't use that palette color.
Let's rather introduce a `statusToastMessage` namespace for that.